e2e/tests/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -319,37 +326,52 @@ test('Example Imagery in Display layout', async ({ page }) => {
     // Verify previous image
     await expect(selectedImage).toBeVisible();
 
-    // Wait 20ms to verify no new image has come in
-    await page.waitForTimeout(21);
+    const imageCount = await page.locator('.c-imagery__thumb').count();
+    await expect.poll(async () => {
+        const newImageCount = await page.locator('.c-imagery__thumb').count();
+
+        return newImageCount;
+    }, {
+        message: "verify that new images still stream in",
+        timeout: 6000
```

Comment:
Just a small nitpick, I've been using 6 * 1000 on all integers just for readability. That's what the playwright team puts in all their examples as well

Reply:
Good catch.